src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -190,36 +190,6 @@ describe('VAOS <RequestedAppointmentDetailsPage>', () => {
     expect(screen.history.push.lastCall.args[0]).to.equal('/');
   });
 
-  it('should go back to requests page when clicking go back to appointments button', async () => {
```

Comment:
We lose this functionality but it wasn't implemented for past/canceled appointments so it seems ok.

The test above this one checks for going back to the appointment list when clicking on the breadcrumbs nav.